app/src/main/java/org/p2p/wallet/send/model/SearchResult.kt

```diff
@@ -4,7 +4,8 @@ import android.os.Parcelable
 import kotlinx.parcelize.Parcelize
 
 sealed class SearchResult(
-    open val address: String
+    open val address: String,
+    open val networkType: NetworkType = NetworkType.SOLANA
```

Comment:
you can remove default value here

Reply:
I've got your point. But the thing is that networkType is changed in AddressOnly results in Bitcoin case, so to not set NetworkType.SOLANA in most places I just used default value